### DISCUSSION TITLE
[Projects Beta] Allow future dates in charts, show milestone dates.

### DISCUSSION THREAD

**Discussion by gdborton** [community/community#13036] [Projects and Issues]
The charting feature has been really useful for visualizing progress toward larger milestones, but it's difficult to keep everything in perspective without being able to see when the milestone is expected to be reached.

Would love to see this chart expand to the selected end date, with markers for milestones.  This would help us see at a glance whether we're in danger of missing and by how much.

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/4172067/159086545-f1ddef62-94ca-4076-bc79-71961993ff1a.png">

**Comment by becca** [comment]
@azenMatt let us know if you have any followup questions about this request. This would really help us across many projects we're using the feature for and we typically use the repo-level Milestone feature or a new custom field `Milestones` that can be used to track issues across repositories in a single project board (unlike the current Milestone feature, unfortunately).

**Reply by azenMatt** [reply]
Hi @becca 👋 this looks like it would be really helpful - I've shared it with the insights team and will let you know if there are any questions 😄

**Reply by danielgomezrico** [reply]
@azenMatt are there any news on this request? :)

**Comment by Jus4Me2** [comment]
> The charting feature has been really useful for visualizing progress toward larger milestones, but it's difficult to keep everything in perspective without being able to see when the milestone is expected to be reached.
> 
> Would love to see this chart expand to the selected end date, with markers for milestones. This would help us see at a glance whether we're in danger of missing and by how much.
> 
> <img alt="image" width="1050" src="https://user-images.githubusercontent.com/4172067/159086545-f1ddef62-94ca-4076-bc79-71961993ff1a.png">
